cmake/utils.cmake

```diff
@@ -274,3 +274,28 @@ function(mayaUsd_copyDirectory target)
 
     endforeach()
 endfunction()
+
+# parse list arguments into a new list separated by ";" or ":"
+function(separate_argument_list listName)
+    if(IS_WINDOWS)
+        string(REPLACE ";" "\;" ${listName} "${${listName}}")
+    else()
+        string(REPLACE ";" ":" ${listName} "${${listName}}")
+    endif()
+    set(${listName} "${${listName}}" PARENT_SCOPE)
+endfunction()
```

Comment:
Created a new utility function to deal with parsing list arguments and ";" or ":" based on Paul's suggestion.